app/views/comments/_comment_form.html.erb

```diff
@@ -3,7 +3,7 @@
   <% commentable = @commentable %>
 <% end %>
 <div class="post comment" id="comment_form_for_<%= commentable.id %>">
-  <%= form_for value_for_comment_form(commentable, comment), :remote => !comment.new_record?, :html => {:id => "comment_for_#{commentable.id}"} do |f| %>
+  <%= form_for value_for_comment_form(commentable, comment), :html => {:id => "comment_for_#{commentable.id}"} do |f| %>
```

Comment:
Can you update the syntax here to `html: { id: "comment_for_#{commentable.id}" }`? Then this should be ready to go!